src/plugins/intel_cpu/src/transformations/cpu_opset/common/op/read_value_with_subgraph.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/op/op.hpp"
+#include "openvino/op/read_value.hpp"
+#include "openvino/op/util/sub_graph_base.hpp"
+#include "transformations/cpu_opset/common/op/submodel.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+class ReadValueWithSubgraphNode : public op::util::MultiSubGraphOp {
```

Comment:
Do we expect it to have multiple subgraphs?
Should be inherited from "ov::op::util::SubGraphOp" if not.

Reply:
Done